seerep-hdf5/seerep-hdf5-fb/src/hdf5-fb-general.cpp

```diff
@@ -126,15 +123,15 @@ void Hdf5FbGeneral::writeBoundingBox2DLabeled(
 
 void Hdf5FbGeneral::writeLabelsGeneral(
     const std::string& datatypeGroup, const std::string& uuid,
-    const flatbuffers::Vector<flatbuffers::Offset<seerep::fb::LabelWithInstance>>* labelsGeneral)
+    const flatbuffers::Vector<flatbuffers::Offset<seerep::fb::LabelWithInstance>>& labelsGeneral)
 {
   std::string id = datatypeGroup + "/" + uuid;
 
-  if (labelsGeneral && !labelsGeneral->size() == 0)
+  if (labelsGeneral.size() != 0)
```

Comment:
same for this: are you sure that this doesn't create a segmentation fault if the labelsGeneral vector in the FB msg isn't set?